public/app/features/teams/TeamGroupSync.tsx

```diff
@@ -163,7 +163,7 @@ export class TeamGroupSync extends PureComponent<Props, State> {
               buttonTitle="Add group"
               proTip={headerTooltip}
               proTipLinkTitle="Learn more"
-              proTipLink="https://docs.grafana.org/auth/enhanced_ldap/"
+              proTipLink="https://grafana.com/docs/grafana/latest/setup-grafana/configure-security/configure-authentication/enhanced-ldap/"
```

Comment:
ldap?
I thought we want to use this page
https://grafana.com/docs/grafana/latest/setup-grafana/configure-security/configure-team-sync/

Reply:
Good catch @brahmnes , fixed in https://github.com/grafana/grafana/pull/65493/commits/e31f05d58e9ae4df92a0d7fddc1543b9ffdc8a9f